p4-16/spec/P4-16-spec.mdk

```diff
@@ -7973,14 +7989,6 @@ parser generator tool.
 
 Several other constant terminals appear in these rules:
 ~ Begin P4Example
-- SHL is <<
-- LE is <=
-- GE is >=
-- NE is !=
-- EQ is ==
-- PP is ++
-- AND is &&
-- OR is ||
```

Comment:
Are these changes intentional?  If so, I guess related to the "Fixed grammar for right shifts" change?

Reply:
Yes, these were removed from the grammar.